LetsStart/src/Model/ProjectList.java

```diff
@@ -26,9 +26,9 @@ public void removeProject(Project project)
   {
     list.remove(project);
   }
-  public void removeProject(int index)
+  public void removeProject(int Id)
   {
-    list.remove(index);
+    list.removeIf(p -> p.getId() == Id);
```

Comment:
Not sure I understand how this line works. What kind of operator is this - "->"?